optimum/pipelines/pipelines_base.py

```diff
@@ -179,7 +179,7 @@ def load_bettertransformer(
     **kwargs,
 ):
     if model_kwargs is None:
-        model_kwargs = {}
+        model_kwargs = {"attn_implementation": "eager"}
```

Comment:
yes i thought about that, should we add a version check ? or increase min version to when the argument was first introduced ?

Reply:
it was introduced in [`4.36.0`](https://github.com/huggingface/transformers/blob/v4.36.0/src/transformers/modeling_utils.py)

Reply:
Increasing the minimum version could work in some cases, it's been a while since we didn't updated the minimum required transformers version so we could definitely upgrade it a bit but here it'll be a bit too much of an increase, so a version check should be enough for now!

Reply:
also it would be awesome to also add tests for lower bound of supported transformers version (that could be added in slow tests) wdyt @IlyasMoutawwakil ?

Reply:
yes increasing the version for one argument is an overkill, I'll add a check !
let's add the min versions tests in another PR, I was planning on refactoring/simplifying the CI a bit once everything is green.

Reply:
done !